lite-lava-dispatcher/Dockerfile

```diff
@@ -2,10 +2,10 @@ ARG image
 
 FROM ${image}
 
-ENV JLINK_DEB="JLink_Linux_V650_x86_64.deb"
+ENV JLINK_DEB="JLink_Linux_V688a_x86_64.deb"
 ENV JLINK_URL="https://www.segger.com/downloads/jlink/${JLINK_DEB}"
 ENV ZEPHYR_HOST_TOOLS="zephyr-sdk-x86_64-hosttools-standalone-0.9.sh"
-ENV ZEPHYR_HOST_TOOLS_URL="https://builds.zephyrproject.org/sdk/0.10.3/${ZEPHYR_HOST_TOOLS}"
+ENV ZEPHYR_HOST_TOOLS_URL="https://builds.zephyrproject.org/sdk/0.11.4/${ZEPHYR_HOST_TOOLS}"
```

Comment:
@galak, Btw, is there a reason why hosttools *still* live under a URL like https://builds.zephyrproject.org/sdk/0.11.4/zephyr-sdk-x86_64-hosttools-standalone-0.9.sh ? What does that "0.9" is supposed to signify?